Sources/LanguageServerProtocol/SupportTypes/SKCompletionOptions.swift

```diff
@@ -15,10 +15,15 @@
 /// **(LSP Extension)**: This is used as part of an extension to the
 /// code-completion request.
 public struct SKCompletionOptions: Codable, Hashable {
+  /// Whether to use server-side filtering or to return all results and let the
+  /// client handle all filtering.
+  public var serverSideFiltering: Bool
+
   /// The maximum number of completion results to return, or `nil` for unlimited.
   public var maxResults: Int?
 
-  public init(maxResults: Int? = 200) {
+  public init(serverSideFiltering: Bool = true, maxResults: Int? = 200) {
+    self.serverSideFiltering = serverSideFiltering
```

Comment:
I removed the `serverSideFiltering` option a few weeks ago. Is it intentional that you add it back?